BackEnd/stripeApi.js

```diff
@@ -0,0 +1,407 @@
+//const stripe = require('stripe')('sk_test_4eC39HqLyjWDarjtT1zdp7dc');
+
+/* Customers API in stripe*/
+
+const createCustomer = (user_id,ticketName,username,email,price,quantity,ticketImg,eventName) => {
+ return stripe.customers.create({
+
+ "id": user._id,
+  "object": "customer",
```

Comment:
Please remove all hardcoded data. All info should be passed via the function parameters